CMSgov/mint-app
# [EASI-2589] Added 'Paused' and 'Canceled' Statuses for Model Plans

# EASI-2589

## Changes and Description

- Added 'Canceled' Model Plan Status
- Added 'Paused' Model Plan Status
- Updated Front-End Status Options
- Updated Front-End test snapshot



## How to test this change

- Create a Model Plan
- Confirm the Model Plan's default status in the database
- Update the Model Plan's status to 'Paused' and 'Canceled'
- Confirm that the database entry for this Model Plan's status is correct

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
